hw/ip/otbn/dv/otbnsim/sim/wsr.py

```diff
@@ -100,6 +102,8 @@ def __init__(self, name: str):
         self._random_value = None  # type: Optional[int]
         self._random_value_read = False
         self.pending_request = False
+        self.prefetch = False
```

Comment:
Do we actually have to distinguish `pending_request` and `prefetch`? (Or can just just use `pending_request` for both?)